### PR TITLE
ELEMENTS-1241: use WriteVersion permission in nuxeo-filters-behavior

### PR DESCRIPTION
**Pull request by mnixo**

### CODE REVIEW

**Comment by mnixo** [comment]
🔵  https://qa.nuxeo.org/jenkins/job/web-ui-pipeline-mnixo/47/

**Review by semisse — DISMISSED** [review]

File: ui/nuxeo-filters-behavior.js [hunk 1]
```diff
@@ -241,7 +241,7 @@ export const FiltersBehavior = {
    * Checks if the document can be tagged.
    */
   isTaggable(doc) {
-    return !this.isProxy(doc) && !doc.isVersion && this.hasPermission(doc, 'Write');
+    return !this.isProxy(doc) && !doc.isVersion && this.hasPermission(doc, 'WriteVersion');
```
Comment (Gabez0r) [inline ui/nuxeo-filters-behavior.js:244]: I'm wondering why do we need  `WriteVersion` to tag a document? :thinking: I would expect us to need `WriteProperties` though, given that the tags are written to the `nxtag:tags`  field.

Comment (mnixo) [reply ui/nuxeo-filters-behavior.js:244]: I agree. I will most likely leave this one out of the scope of this ticket. If it is reported as problem in the future it should be an easy fix.

Comment (Gabez0r) [reply ui/nuxeo-filters-behavior.js:244]: I agree.

**Review by semisse — APPROVED** [review]

**Review by Gabez0r — APPROVED** [review]

**Pull request merged**